scripts/raspberry_pi_setup.sh

```diff
@@ -1,18 +1,32 @@
 #!/bin/bash
 
-SCRIPT_DIR=`dirname "${BASH_SOURCE[0]}"`
-if [ -e "$SCRIPT_DIR/python.sh" ]; then
-    KALITE_DIR=$SCRIPT_DIR/../kalite
-else
-    KALITE_DIR=$SCRIPT_DIR/kalite
+if [ `whoami` == "root" ]
+then
+    echo "Cannot run as root!"
+    exit 1
+fi
+
+# Create configuration directory in case we never ran before
+mkdir -p ~/.kalite
+
+USER_CONFIG=~/.kalite/settings.py
+
+if ! grep -Pq "^from kalite.project.settings.raspberry_pi" $USER_CONFIG
+then
+    echo "Changing user config to use raspberry pi settings: $USER_CONFIG"
+    echo "from kalite.project.settings.raspberry_pi import *" > $USER_CONFIG
 fi
 
-#we_are_rpi=`"$current_dir/get_setting.sh" package_selected\(\"RPi\"\)`
-#if [ $we_are_rpi != "True" ]; then
-#    echo "Error: we aren't configured as a Raspberry Pi, cannot continue"
-#    read WAITING
-#    exit
-#fi
```

Comment:
outdated, look at the above code -- it forces the user's settings to use the raspberry_pi settings module instead.